content/posts/2018-03-18-dependency-upgrade.md

```diff
@@ -0,0 +1,145 @@
+---
+title: "The Art of a Dependency Upgrade"
+description: "A walk to a technical marvel without breaking a leg or two."
+date: 2018-03-18
+slug: art-of-a-dependency-upgrade
+---
+
+[RxJava](https://github.com/ReactiveX/RxJava) 1.+ reaches EOL on March 31, 2018,
```

Comment:
Change 1.+ to `1.x`? That's more commonly used and reads better imo

Reply:
Yep, let’s do it.